blackbox/docs/sql/reference/create_table.txt

```diff
@@ -181,17 +181,16 @@ The optional CLUSTERED clause specifies how a table should be
 distributed accross a cluster.
 
 :num_shards: specifies the number of shards a table is stored in.
-    Must be greater than 0. If no number is specified the default number
-    of shards is applied. The calculated value of ``num_shards`` is equal to
-    twice the number of data-nodes in the cluster
+    Must be greater than 0. If no number is specified a default value is
+    applied. The calculated value of ``num_shards`` is equal to twice the
```

Comment:
... Must be greater than 0. If not provided the number of shards is calculated based on the number of currently active data nodes with the following formula: